src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/curatedcontent/CuratedMainPagePageObject.java

```diff
@@ -65,6 +68,15 @@ public boolean isWikiaStatsContainerVisible() {
     return isCuratedElementVisible(wikiaStatsContainer);
   }
 
+  public boolean isMainPagePadSlotInDOM() {
+    try {
+      wait.forElementPresent(mainPagePadSlot, false);
+      return true;
+    } catch (TimeoutException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)